src/main/java/org/seatsurfer/domain/Admin.java

```diff
@@ -0,0 +1,79 @@
+package org.seatsurfer.domain;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+@Entity
+@Table(
+        name = "admins",
+        uniqueConstraints = {
+            @UniqueConstraint(
+                name = "user_name_unique",
+                columnNames = "name"
+            ),
+            @UniqueConstraint(
+                name = "user_email_unique",
+                columnNames = "email"
+            ),
+            @UniqueConstraint(
+                name = "user_password_unique",
+                columnNames = "password"
+            )
+        }
+)
+public class Admin {
+    @Id
+    @Column(
+            name = "id",
+            updatable = false
+    )
+    private Long id;
+
+    @Column(
+            name = "name",
+            nullable = false,
+            columnDefinition = "TEXT"
+    )
+    private String name;
+
+    @Column(
+            name = "email",
+            nullable = false,
+            columnDefinition = "TEXT"
+    )
+    private String email;
+
+    @Column(
+            name = "password",
+            nullable = false,
+            columnDefinition = "TEXT"
```

Comment:
I am not sure if it helps to mention columnDefinition, from documentation "Defaults to the generated SQL to create a column of the inferred type."

Reply:
Yes, i removed columnDefinition.